main.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Please do not keep main.go in the root directory since we do not have only one CLI in the repo.